frontend/src/scenes/insights/ActionFilter/ActionFilterRow.tsx

```diff
@@ -281,16 +306,42 @@ function MathSelector({ math, index, onMathSelect, areEventPropertiesNumericalAv
     )
 }
 
-function MathPropertySelector(props) {
+interface MathPropertySelectorProps {
+    name: string | null
+    math?: string
+    mathProperty?: string
+    index: number
+    onMathPropertySelect: (index: number, value: string) => any
+    properties: EventProperty[]
+}
+
+interface SelectOptionType {
```

Comment:
Seems like this could probably be useful for other components, maybe worth abstracting

Reply:
Turns out I never needed to access `children` or `data-attr`, but I'll extract this into a new interface called `SelectOptionWithChildren` anyway.